src/main/java/kr/sesac/aoao/server/dino/repository/DinoInfoEntity.java

```diff
@@ -0,0 +1,39 @@
+package kr.sesac.aoao.server.dino.repository;
+
+import java.util.List;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import kr.sesac.aoao.server.global.entity.BaseEntity;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Builder
+@Getter
+@Table(name = "dino_info")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+public class DinoInfoEntity extends BaseEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long Lv;
+
+	@Column(nullable = false)
+	private int all_exp;
+
+	@Column(nullable = false, length = 30)
+	private String lv_name;
```

Comment:
```suggestion
	@Column(nullable = false)
	private int allExp;

	@Column(nullable = false, length = 30)
	private String lvName;
```

다이노 테이블이 실제 컬럼명이 `all_exp` 와 `lv_name` 이 맞을까요?? 맞다면 다음과 같이 자바 컨벤션에 맞게 변수명을 지정한다면 JPA 에서 자동으로 매칭해줍니다! 완전히 다른 이름 (lv_name -> levelName) 으로 변경하고 싶은 경우에만 따로 지정하면 됩니당